Source/ASTextNode.mm

```diff
@@ -230,6 +230,8 @@ - (void)dealloc
 {
   CGColorRelease(_shadowColor);
 
+  // TODO: This may not be necessary post-iOS-9 when most UIKit assign APIs
+  // were changed to weak.
```

Comment:
This file was already modified post-Texture, so this change would have triggered a danger warning without the hack added in this diff.